### DISCUSSION TITLE
Read `SkipRemainingOnFailureCount` while running pester?

### DISCUSSION THREAD

**Discussion by schittli** [pester/Pester#2261] [Q&A] [answered]
Good evening

Pester counts the number of failures in `SkipRemainingOnFailureCount` which is a pretty interesting value. Is it possible to read it while running pester?

Thanks a lot for any help!,
kind regards,
Thomas

**Comment by fflaten** [comment, marked as answer]
Only by using an internal API which I'd recommend you avoid as it may change without warning.
May I ask when it would be useful?

By using `-Passthru` or `Run.Passthru` when you invoke Pester, you can find the skipped tests in the result-object

```powershell
Invoke-Pester myfile.tests.ps1 -Passthru
$r.Skipped | ? ErrorRecord -match 'Run.SkipRemainingOnFailure' | measure
```
